cabal.project

```diff
@@ -76,10 +76,6 @@ if os(windows)
 package bitvec
   flags: -simd
 
--- See the note on nix/agda-packages.nix for why this is here.
```

Comment:
Obsolete: `adga-pacakges.nix` uses `cabalProjectLocal` to add `ieee` and `filemanip`